bigtable/admin.go

```diff
@@ -1034,6 +1036,11 @@ func (iac *InstanceAdminClient) Clusters(ctx context.Context, instanceID string)
 			StorageType: storageTypeFromProto(c.DefaultStorageType),
 		})
 	}
+	if len(res.FailedLocations) > 0 {
```

Comment:
can we add a test?

Reply:
+1, this would help.

Reply:
Hm-m, as we're not very able to simulate "failed locations" error on real data (at least I don't know how this can be done), I think a test can be written with kind of a mock for `ListClusters()` method. But the mock is big, as there is an interface implementation restriction, and the interface describes a lot of methods. By the way, I see that emulator doesn't support most part of instance and cluster related methods. Should we implement them? Doesn't seem to be very hard.